### ISSUE TITLE
Footer image missing alt text. 

### ISSUE THREAD

**Issue by MelSumner** [emberjs/website#3057]
**Expected:**
Images must have alternate text when they are not in a presentation role. 

**Actual:**
2 images in footer do not have alt text.

**To Fix:**
Since this image is used as the only element in the link, it requires alt text. 
- Add alt="heroku" to the img element inside the link to Heroku. 
- add alt="fastly" to the img element inside the link to Fastly. 

**Screenshot:** 
![issue_footer-img](https://user-images.githubusercontent.com/4587451/32147483-09810960-bcb6-11e7-80fd-432ce9e928e8.JPG)
